resources/views/api/includes/contents/app.blade.php

```diff
@@ -1,22 +1,49 @@
-<!-- top nav -->
-<div class="row">
+@if ( Auth::check() )
+
+        <!-- top nav -->
     @include("api.includes.sections.top_nav_user")
```

Comment:
**Code Quality - Nitpick**

Repetition. This is included in the if and else. Shouldn't the view be included outside of the if & else block?